### ISSUE TITLE
Better date checking required

### ISSUE THREAD

**Issue by bnmnetp** [RunestoneInteractive/RunestoneServer#913]
Traceback (most recent call last): 
```
File "/home/bnmnetp/webapps/uwsgi_runestone/web2py/gluon/restricted.py", line 227, in restricted 
exec ccode in environment 
File "/home/bnmnetp/webapps/uwsgi_runestone/web2py/applications/runestone/controllers/admin.py", line 1341, in <module> 
File "/home/bnmnetp/webapps/uwsgi_runestone/web2py/gluon/globals.py", line 417, in <lambda> 
self._caller = lambda f: f() 
File "/home/bnmnetp/webapps/uwsgi_runestone/web2py/gluon/tools.py", line 4241, in f 
return action(*a, **b) 
File "/home/bnmnetp/webapps/uwsgi_runestone/web2py/applications/runestone/controllers/admin.py", line 621, in admin 
date = date.strftime("%m/%d/%Y") 
ValueError: year=17 is before 1900; the datetime strftime() methods require year >= 1900
```

We should either catch this in the javascript or in the controller.  Not sure which option is better at the moment, but @conzty01 and @gardhi01 could look at the options and make a recommendation.

**Comment by gardhi01** [comment]
We looked this over, and we discovered the problem occurs when an instructor enters a term_start_date in the mm/dd/yy format as opposed to the mm/dd/yyyy format. This causes the database to record the term_start_date as 00yy instead of 20yy.  We think catching this in the javascript would be better, perhaps by using a datepicker that only accepts yyyy or using a datepicker that will format yy into 20yy.
